stdlib/3/_winapi.pyi

```diff
@@ -63,13 +63,13 @@ def GetStdHandle(std_handle: int) -> int: ...
 def GetVersion() -> int: ...
 def OpenProcess(desired_access: int, inherit_handle: bool, process_id: int) -> int: ...
 def PeekNamedPipe(handle: int, size: int = ...) -> Union[Tuple[int, int], Tuple[bytes, int, int]]: ...
-def ReadFile(handle: int, size: int, overlapped: Union[int, bool] = ...) -> Tuple[bytes, int]: ...
+def ReadFile(handle: int, size: int, overlapped: Union[int, bool] = ...) -> Tuple[Overlapped, int]: ...
```

Comment:
This will return `Tuple[bytes, int]` if overlapped is not provided or `False`. An `@overload` seems like the best solution.